backend/package-lock.json

```diff
@@ -1,2375 +1,8 @@
 {
   "name": "backend",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
```

Comment:
what version of npm and node are you using?)

Reply:
npm: '6.14.15', node: '14.18.1'

Reply:
> what version of npm and node are you using?)
npm: '6.14.15', node: '14.18.1'

Reply:
I updated my npm version and lockfileVersion right now is 3